### PR TITLE
🔀🔖 4.7.5

### PR DESCRIPTION
**Pull request by jxn-30**
**Releasenotes**
* 🐛 [blf] fix buggy design in bigMapMode
* 🐛 [ecl/sm] fix equipment not being alarmed on sorted alarming
* 🐛 [em/bc] fix protocol of alliance buildings
* 🐛 [redesign/profile] fix issue with negative latitudes/longitudes on map
* 🐛 [settings] prevent downloading invalid export-file
* 💄 [redesign] add a dark loading screen in dark mode
* 💄 [redesign/messages] improve height for large profile pics
* ✨ [ecw] option for permanent search bar #2308
* ✨ [emv/arrHighlight] Do not include fire alarm system text
* ✨ [mH] option to not load on patient transfer missions
* ✨ [sap] add a `{{share}}` variable using timestamp of first mission response text otherwise now

---
**Releasenotes for translations**
```md
* 🐛 [$m('buildingListFilter')] fix buggy design in bigMapMode
* 🐛 [$s('extendedCallList.sortMissions')] fix equipment not being alarmed on sorted alarming
* 🐛 [$s(!'extendedMap.buildingComplexes')] fix protocol of alliance buildings
* 🐛 [$m('redesign')/profile] fix issue with negative latitudes/longitudes on map
* 🐛 [settings] prevent downloading invalid export-file
* 💄 [$m('redesign')] add a dark loading screen in dark mode
* 💄 [$m('redesign')/messages] improve height for large profile pics
* ✨ [$m('extendedCallWindow')] option for permanent search bar #2308
* ✨ [$s('extendedCallWindow.arrMatchHighlight')] Do not include fire alarm system text
* ✨ [$m('missionHelper')] option to not load on patient transfer missions
* ✨ [$m('shareAlliancePost')] add a `{{share}}` variable using timestamp of first mission response text otherwise now
```

### CODE REVIEW

File: src/modules/dashboard/components/vehicle-list.vue [hunk 1]
```diff
@@ -274,11 +274,11 @@ export default Vue.extend<
             let missionName =
                 document.querySelector<HTMLAnchorElement>(
                     `#mission_caption_${id}`
-                )?.innerText ?? '';
```
Comment (Cr4zyc4k3) [inline src/modules/dashboard/components/vehicle-list.vue:277]: In which case is textContent better than innerText?

File: src/modules/redesign/parsers/profile.ts [hunk 2]
```diff
@@ -146,7 +148,7 @@ export default <RedesignParser<ProfileWindow>>(({ LSSM, doc, href = '' }) => {
                                 prop => [
                                     prop.key.value,
                                     prop.value.type === 'UnaryExpression'
-                                        ? parseInt(
+                                        ? parseFloat(
```
Comment (Cr4zyc4k3) [inline src/modules/redesign/parsers/profile.ts:151]: Are there really awards with floats? Or just prevention?

Comment (jxn-30) [reply src/modules/redesign/parsers/profile.ts:151]: This is the detection of building latitudes and longitudes. As those can be floats, this is a bug fix.

File: src/modules/dashboard/components/vehicle-list.vue (hunk 1 above)
Comment (jxn-30) [reply src/modules/dashboard/components/vehicle-list.vue:277]: there are multiple advantages such as better performance and more predictable behaviour.
It is enforced by ESLint-Rule [unicorn/prefer-dom-node-text-content](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-dom-node-text-content.md) to prefer `.textContent` over `.innerText`

**Review by Cr4zyc4k3 — APPROVED** [review]

**Review by Suchty112 — APPROVED** [review]

**Pull request merged**